### DISCUSSION TITLE
Extend "National code" by other national annexes for EC (Hungarian NA, ... )

### DISCUSSION THREAD

**Discussion by scia-nv** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#13] [Ideas]
Add Hungarian National annex as new value for enum "National code" in Model sheet.

Proposed value = EC-MSZ-EN (Hungarian NA)

**Comment by dschuermans** [comment]
FYI, this would be patching the 2.0.0 spec.
Thus the new version should be 2.0.1 (unless ofc you really want to wait with this change until the 2.1.0 spec is ready to go)

**Comment by robinblazek** [comment]
Add SIA Standard as new value for enum "National code" in Model sheet.

Proposed value = SIA 26x

**Comment by MartinSalak** [comment]
National annexes we are missing in SAF:

EC-ТКП-EN (Belarusian NA)
EC-BAS-EN (Bosnian and Herzegovinian NA)
EC-BDS-EN (Bulgarian NA)
EC-EVS-EN (Estonian NA)
EC-MSZ-EN (Hungarian NA)
EC-IST-EN (Icelandic NA)
EC-LVS-EN (Latvian NA)
EC-LST-EN (Lithuanian NA)
EC-MSA-EN (Maltese NA)
EC-MKC-EN (North Macedonian NA)
EC-NP-EN (Portuguese NA)
EC-SRPS-EN (Serbian NA)
EC-SN-EN (Swiss NA)
EC-TS-EN (Turkish NA)

**Comment by dschuermans** [comment]
What in terms of backward compatibility?

If a SAF file with one of these new national codes is imported as or exported as SAF 2.0?
Can we put it to "EC-Standard-EN"? Or should the operation be blocked until the national code value is manually fixed?

**Reply by MartinSalak** [reply]
We believe the best solution would be to export (eventually import) it as EC-Standard-EN.

**Comment by sonkaste17** [comment]
@msalak-scia Can you add them to the latest documentation, please?

**Comment by MartinSalak** [comment]
National code extended in the latest version of documentation which will be released as a next SAF version. Take a look here:
https://www.saf.guide/en/latest/getting-started/project-and-model-specifications/README.html

All comments and notes are welcomed.
